google/cloud/forseti/enforcer/enforcer.py

```diff
@@ -99,17 +83,18 @@ def initialize_batch_enforcer(global_configs, concurrent_threads,
     """Initialize and return a BatchFirewallEnforcer object.
 
     Args:
-      global_configs (dict): Global configurations.
-      concurrent_threads: The number of parallel enforcement threads to execute.
-      max_write_threads: The maximum number of enforcement threads that can be
-          actively updating project firewalls.
-      max_running_operations: The maximum number of write operations per
-          enforcement thread.
-      dry_run: If True, will simply log what action would have been taken
-          without actually applying any modifications.
+        global_configs (dict): Global configurations.
+        concurrent_threads (str): The number of parallel enforcement threads to
+            execute.
+        max_write_threads (str): The maximum number of enforcement threads that
+            can be actively updating project firewalls.
+        max_running_operations (str): The maximum number of write operations per
+            enforcement thread.
+        dry_run (boolean): If True, will simply log what action would have been
+            taken without actually applying any modifications.
 
     Returns:
-      A BatchFirewallEnforcer instance.
+        class: BatchFirewallEnforcer
```

Comment:
Just wanted to clarify, shouldn't this be something like `BatchFirewallEnforcer: A BatchFirewallEnforcer instance.`

Reply:
Yup, fixed.